packages/@okta/vuepress-site/docs/guides/sign-in-with-microsoft/configure-idp/index.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Configure Facebook as an Identity Provider in Okta
+title: Configure Microsoft as an Identity Provider in Okta
 ---
 
 2.1. Sign in to your Okta org.
 
 2.2. On the main page, click on the **Admin** button in the upper right.
 
-2.3. Hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.3. If you use the Developer Console, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**. If you use the Classic UI, hover over **Security** then click on **Identity Providers**.
 
 2.4. On the Identity Providers page, click on **Add Identity Provider** > **Add Microsoft**.
 
 * **Name:** We suggest using the name you would expect to see on a button, something like "Log in to Microsoft".
-* **Client Id:** Paste in the App ID that you got from Microsoft in step 1.3 above.
-* **Client Secret:** Paste in the App Secret that you got from Microsoft in step 1.3 above.
+* **Client Id:** Paste in the App ID that you got from Microsoft in step 1.3.
+* **Client Secret:** Paste in the App Secret that you got from Microsoft in step 1.3.
```

Comment:
**Client Secret:** Paste the App Secret that you got from Microsoft in step 1.3.